gradio/themes/app.py

```diff
@@ -1,8 +1,7 @@
 import time
 
-from theme_dropdown import create_theme_dropdown  # noqa: F401
-
 import gradio as gr
+from gradio.themes.utils.theme_dropdown import create_theme_dropdown
```

Comment:
Let's delete the `theme_dropdown.py` from the space that's pushed up (in themes/base.py)

```
            CommitOperationAdd(
                path_in_repo="theme_dropdown.py",
                path_or_fileobj=str(
                    Path(__file__).parent / "utils" / "theme_dropdown.py"
                ),
            ),
```

Reply:
Done.